conf/layer.conf

```diff
@@ -0,0 +1,10 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH .= ":${LAYERDIR}"
+
+# We have recipes-* directories, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+	${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "validation"
+BBFILE_PATTERN_validation = "^${LAYERDIR}/"
+BBFILE_PRIORITY_validation = "6"
```

Comment:
Please add LAYERSERIES_COMPAT_validation to indicate the versions you're compatible with: https://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html#var-LAYERSERIES_COMPAT